schema.sql

```diff
@@ -53,3 +53,7 @@ CREATE TABLE visits (
     visit_date DATE,
     PRIMARY KEY (animal_id, vet_id, visit_date)
 );
+
+ CREATE INDEX ON visits(animal_id);
+
+ ANALYZE visits;
```

Comment:
- [ ] Good Job for **implementing indexes for the expected solution**  but kindly also make sure that you added an indexes for visits table with the `vet_id` and also for for owners table with `email` column just like the below sample.

```sql
CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);
```